frontend/cypress/integration/featureFiles/workload_logs.feature

```diff
@@ -23,13 +23,13 @@ Feature: Workload logs tab
 
   @bookinfo-app
```

Comment:
This is called bookinfo-logger but is the life-cycle tied to the bookinfo app?  The test requires bookinfo, but I don't see how the logger is tied to it. It seems separate, in a different namespace, and not coming and going with bookinfo itself. Maybe I am confused.

Reply:
You're not, used to be tight, not anymore, I'll remove it.